sonar-project.properties

```diff
@@ -0,0 +1,2 @@
+sonar.projectKey=City-of-Helsinki_linkedevents
+sonar.organization=city-of-helsinki
```

Comment:
We could/should add `sonar.python.version=3.9` here?

Reply:
Yes 👍